SurveyTests/Sources/Dummy/APIToken+Dummy.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  APIToken+Dummy.swift
+//  SurveyTests
+//
+//  Created by Khanh on 08/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+@testable import Survey
+
+extension APIToken {
+
+    static let dummy = APIToken(id: "1",
+                                type: "token",
+                                accessToken: "accessToken",
+                                tokenType: "tokenType",
+                                expiresIn: 900,
+                                refreshToken: "refreshToken",
+                                createdAt: 1_888_882)
```

Comment:
Please break new line at each brackets. Use auto format after line break.
```suggestion
    static let dummy = APIToken(
                                id: "1",
                                type: "token",
                                accessToken: "accessToken",
                                tokenType: "tokenType",
                                expiresIn: 900,
                                refreshToken: "refreshToken",
                                createdAt: 1_888_882
                                )
```